docs/tutorials/advanced/reading_and_writing_interactions.rst

```diff
@@ -7,70 +7,108 @@ a :code:`filename` argument to the :code:`play` method of a tournament::
+    2,7,Bully,Win-Stay Lose-Shift,DDCD,CDCC
+    3,3,Cooperator,Cooperator,CCCC,CCCC
+    3,3,Cooperator,Cooperator,CCCC,CCCC
+    3,4,Cooperator,Defector,CCCC,DDDD
+    3,4,Cooperator,Defector,CCCC,DDDD
+    3,5,Cooperator,Suspicious Tit For Tat,CCCC,DCCC
+    3,5,Cooperator,Suspicious Tit For Tat,CCCC,DCCC
+    3,6,Cooperator,Tit For Tat,CCCC,CCCC
+    3,6,Cooperator,Tit For Tat,CCCC,CCCC
+    3,7,Cooperator,Win-Stay Lose-Shift,CCCC,CCCC
+    3,7,Cooperator,Win-Stay Lose-Shift,CCCC,CCCC
+    4,4,Defector,Defector,DDDD,DDDD
+    4,4,Defector,Defector,DDDD,DDDD
+    4,5,Defector,Suspicious Tit For Tat,DDDD,DDDD
+    4,5,Defector,Suspicious Tit For Tat,DDDD,DDDD
+    4,6,Defector,Tit For Tat,DDDD,CDDD
+    4,6,Defector,Tit For Tat,DDDD,CDDD
+    4,7,Defector,Win-Stay Lose-Shift,DDDD,CDCD
+    4,7,Defector,Win-Stay Lose-Shift,DDDD,CDCD
+    5,5,Suspicious Tit For Tat,Suspicious Tit For Tat,DDDD,DDDD
+    5,5,Suspicious Tit For Tat,Suspicious Tit For Tat,DDDD,DDDD
+    5,6,Suspicious Tit For Tat,Tit For Tat,DCDC,CDCD
+    5,6,Suspicious Tit For Tat,Tit For Tat,DCDC,CDCD
+    5,7,Suspicious Tit For Tat,Win-Stay Lose-Shift,DCDD,CDDC
+    5,7,Suspicious Tit For Tat,Win-Stay Lose-Shift,DCDD,CDDC
+    6,6,Tit For Tat,Tit For Tat,CCCC,CCCC
+    6,6,Tit For Tat,Tit For Tat,CCCC,CCCC
+    6,7,Tit For Tat,Win-Stay Lose-Shift,CCCC,CCCC
+    6,7,Tit For Tat,Win-Stay Lose-Shift,CCCC,CCCC
+    7,7,Win-Stay Lose-Shift,Win-Stay Lose-Shift,CCCC,CCCC
+    7,7,Win-Stay Lose-Shift,Win-Stay Lose-Shift,CCCC,CCCC
 
 The columns of this file are of the form:
 
 1. Index of first player
 2. Index of second player
 3. Name of first player
 4. Name of second player
-5. All further columns are interactions written in a compressed format.
+5. History of play of the first player
+6. History of play of the second player
 
-Alternator versus TitForTat has the following interactions: :code:`CCDCCDDC`:
+Note that depending on the order in which the matches have been played, the rows
+could also be in a different order.
+
+Alternator versus TitForTat has the following interactions: :code:`CCDC, CDCD`:
 
 - First turn: :code:`C` versus :code:`C` (the first two letters)
 - Second turn: :code:`D` versus :code:`C` (the second pair of letters)
 - Third turn: :code:`C` versus :code:`D` (the third pair of letters)
 - Fourth turn: :code:`D` versus :code:`C` (the fourth pair of letters)
 
-This can be transformed in to the usual interactions using the
-:code:`interactions_util.string_to_interactions` function:
+This can be transformed in to the usual interactions by zipping:
 
-    >>> string = 'CCDCCDDC'
-    >>> axl.interaction_utils.string_to_interactions(string)
-    [('C', 'C'), ('D', 'C'), ('C', 'D'), ('D', 'C')]
+    >>> list(zip("CCDC", "CDCD"))
+    [('C', 'C'), ('C', 'D'), ('D', 'C'), ('C', 'D')]
 
 This should allow for easy manipulation of data outside of the capabilities
 within the library, but it is also possible to generate a standard result set
```

Comment:
If we go with my suggestion ([on the play method of the tournament](https://github.com/Axelrod-Python/Axelrod/pull/572/files#diff-dad184e860a161b8b929a7ebbfd21c9cR67)) of the possibility of not creating the results set (`play(build_results=False)`) then we could add that here (just briefly mentioning that it's possible and might be of interest due to high memory consumption for large tournaments).